models/r1_rover/r1_rover.sdf

```diff
@@ -131,39 +131,51 @@
     <link name='lb_wheel_link'>
       <pose>-0.15 0.16317 0.0215 0 -0 0</pose>
       <inertial>
-        <pose>0 0 0 0 -0 0</pose>
+        <pose>0 0 0 1.57079632679 0 0</pose>
         <mass>0.414</mass>
         <inertia>
-          <ixx>0.00097298</ixx>
+          <ixx>0.00068682</ixx>
           <ixy>0</ixy>
           <ixz>0</ixz>
-          <iyy>0.00132536</iyy>
+          <iyy>0.00068682</iyy>
           <iyz>0</iyz>
           <izz>0.00097299</izz>
         </inertia>
       </inertial>
       <collision name='lb_wheel_link_collision'>
+        <pose>0 0 0 1.57079632679 0 0</pose>
         <geometry>
-          <mesh>
-            <scale>1 1 1</scale>
-            <uri>model://r1_rover/meshes/wheel_link_simple.STL</uri>
-          </mesh>
+          <cylinder>
+            <radius>0.0686</radius>
+            <length>0.0762</length>
+          </cylinder>
         </geometry>
         <max_contacts>1</max_contacts>
         <surface>
           <friction>
+            <torsional>
+              <coefficient>0.25</coefficient>
+              <use_patch_radius>1</use_patch_radius>
+              <surface_radius>0.0686</surface_radius>
+            </torsional>
             <ode>
-              <mu>200</mu>
-              <mu2>100</mu2>
-              <fdir1>0 0 1</fdir1>
+              <mu>1.0</mu>
+              <mu2>0.45</mu2>
+              <fdir1>1 0 0</fdir1>
               <slip1>0.0</slip1>
               <slip2>0.0</slip2>
             </ode>
           </friction>
+          <bounce>
+            <restitution_coefficient>0</restitution_coefficient>
+            <threshold>1e6</threshold>
+          </bounce>          
           <contact>
             <ode>
-              <min_depth>0.005</min_depth>
-              <kp>1e8</kp>
+              <min_depth>0.001</min_depth>
+              <mav_vel>0.0</mav_vel>
```

Comment:
This results in a failure when running on gazebo 9.13.0 
```
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
```

Reply:
@Jaeyoung-Lim thanks, I just fixed the typo in the ode (max_vel) and made it so people still stuck to gazebo 9 can use it by bumping it down to 1.6 ;-) .